src/lib/journey/journey.store.ts

```diff
@@ -102,7 +108,7 @@ export function initialize(initOptions?: StepOptions): JourneyStore {
 
   let stepNumber = 0;
 
-  async function next(prevStep: StepTypes = null, nextOptions?: StepOptions, resumeUrl?: string) {
+  async function next(prevStep?: StepTypes, nextOptions?: StartOptions, resumeUrl?: string) {
```

Comment:
typescript did not like the =`null` as it was a union of `StepTypes | undefined` but I just made it a nullable type and it appeased the type checker.